client/src/app/+video-channels/video-channels-routing.module.ts

```diff
@@ -37,6 +48,16 @@ const videoChannelsRoutes: Routes = [
         }
       }
     ]
+  },
+  {
+    path: ':videoChannelName/update',
```

Comment:
`update/:videoChannelName` for consistency with the previous route

Reply:
Then it will collide if there's a channel named "update"?